website/pages/en/index.js

```diff
@@ -74,115 +71,211 @@ HomeSplash.propTypes = {
   language: React.PropTypes.string
 };
 
+const TldrSection = () => (
+  <div className="tldrSection productShowcaseSection lightBackground paddingTop paddingBottom">
+    <Container>
+      <div style={{ display: "flex", flexFlow: "row wrap" }}>
+        <div style={{ flex: "1 1 auto" }}>
+          <h2>What is Prettier?</h2>
+          <ul>
+            <li>An opinionated code formatter</li>
+            <li>Supports many languages</li>
+            <li>Integrates with the most editors</li>
+            <li>Has few options</li>
+          </ul>
+        </div>
+        <div style={{ flex: "1 1 auto" }}>
+          <h2>Why?</h2>
+          <ul>
+            <li>You press save and code is formatted</li>
+            <li>No need to discuss style in code review</li>
+            <li>Save you time and energy</li>
+          </ul>
+        </div>
+      </div>
+    </Container>
+  </div>
+);
+
+const LanguagesSection = () => (
+  <div
+    className="languagesSection productShowcaseSection paddingTop paddingBottom"
+    style={{ textAlign: "center" }}
+  >
+    <Container>
+      <h2 style={{ margin: 0 }}>Language Support</h2>
+      <GridBlock
+        align="center"
+        contents={siteConfig.supportedLanguages.map(language => ({
+          title: language.name,
+          image: language.image,
+          imageAlign: "top",
+          content: language.variants.join("\n\n")
+        }))}
+        layout="fourColumn"
+      />
+    </Container>
+  </div>
+);
+
+const Editor = ({ content = "", image, name }) => (
+  <div style={{ display: "flex", width: "235px", padding: "20px" }}>
+    <img src={image} style={{ width: "100px", height: "100px" }} />
```

Comment:
We need to find a white Markdown logo for this, since black on dark blue looks bad.